### PR TITLE
nxos storage modules bug fixes

### PR DESCRIPTION
**Pull request by srbharadwaj**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by srbharadwaj** [comment]
Fixes following issues in ansible/ansible
https://github.com/ansible/ansible/issues/68343
https://github.com/ansible/ansible/issues/68146
https://github.com/ansible/ansible/issues/68144

**Comment by srbharadwaj** [comment]
@NilashishC can you please let me know what are these failures?

**Comment by NilashishC** [comment]
@srbharadwaj Looks like there are linting errors. We run black in check mode against the collection. [Here](https://dashboard.zuul.ansible.com/t/ansible/build/df93e9831d3d40f2992516df4bf687cb/log/job-output.txt#771-773) is the invocation.

This resulted in 5 lines that needed to be reformatted.

(a) https://dashboard.zuul.ansible.com/t/ansible/build/df93e9831d3d40f2992516df4bf687cb/log/job-output.txt#717-719
(b) https://dashboard.zuul.ansible.com/t/ansible/build/df93e9831d3d40f2992516df4bf687cb/log/job-output.txt#771-773

You can pip install black and run it against the files you change before pushing the PR.

File: tests/unit/modules/network/nxos/storage/test_nxos_zone_zoneset.py [hunk 1]
```diff
@@ -705,3 +695,71 @@ def test_zoneset_activate_deactivate_2(self):
...
+                "terminal dont-ask",
+                "zone name zv221 vsan 221",
+                "no member pwwn 21:01:00:1b:32:a1:c0:a8",
+                "no member pwwn 50:06:01:6a:47:e4:6e:59",
+                "zone commit vsan 221",
+                "no terminal dont-ask",
+            ],
+        )
+
+    # def test_bug_from_active_zoneset_add(self):
```
Comment (NilashishC) [inline tests/unit/modules/network/nxos/storage/test_nxos_zone_zoneset.py:733]: This unit test seems to be commented out. Can you add it back?

Comment (srbharadwaj) [reply tests/unit/modules/network/nxos/storage/test_nxos_zone_zoneset.py]: the testcase was changed, added the changed test case

**Comment by srbharadwaj** [comment]
@NilashishC Can you please let me know what is the problem this time? black ran fine for me for all the .py files

**Comment by NilashishC** [comment]
@srbharadwaj so the black command we run is sets the [max line length to 79](https://object-storage-ca-ymq-1.vexxhost.net/v1/a0b4156a37f9453eb4ec7db5422272df/ansible_23/23/37c142267c8bb84ed40a5edcd339faba3016e382/check/ansible-tox-linters/34e4cd9/job-output.html#l555), which I think is stricter than the default. Can you try running `black -v -l79` on the files and then push?

**Comment by NilashishC** [comment]
@srbharadwaj Two files were not linted properly - https://dashboard.zuul.ansible.com/t/ansible/build/5ffc11d0e5de4eca9be2300d198fcd9c/log/job-output.txt#811-813

Please run black as with the same line length as Zuul does without the "--check": https://dashboard.zuul.ansible.com/t/ansible/build/5ffc11d0e5de4eca9be2300d198fcd9c/log/job-output.txt#555

**Comment by srbharadwaj** [comment]
> @srbharadwaj Two files were not linted properly - https://dashboard.zuul.ansible.com/t/ansible/build/5ffc11d0e5de4eca9be2300d198fcd9c/log/job-output.txt#811-813
> 
> Please run black as with the same line length as Zuul does without the "--check": https://dashboard.zuul.ansible.com/t/ansible/build/5ffc11d0e5de4eca9be2300d198fcd9c/log/job-output.txt#555

I did run back with same line length without the "--check" .. `(mfvenv) SUBHARAD-M-34S1:mfvenv subharad$ black -v -l 79 /Users/subharad/ansible_project/collections/ansible_collections/cisco/nxos/tests/unit/modules/network/nxos/storage/
/Users/subharad/ansible_project/collections/ansible_collections/cisco/nxos/tests/unit/modules/network/nxos/storage/__init__.py wasn't modified on disk since last run.
/Users/subharad/ansible_project/collections/ansible_collections/cisco/nxos/tests/unit/modules/network/nxos/storage/test_nxos_devicealias.py wasn't modified on disk since last run.
/Users/subharad/ansible_project/collections/ansible_collections/cisco/nxos/tests/unit/modules/network/nxos/storage/test_nxos_vsan.py wasn't modified on disk since last run.
/Users/subharad/ansible_project/collections/ansible_collections/cisco/nxos/tests/unit/modules/network/nxos/storage/test_nxos_zone_zoneset.py wasn't modified on disk since last run.
All done! ✨ 🍰 ✨
4 files left unchanged.
(mfvenv) SUBHARAD-M-34S1:mfvenv subharad$ `

**Comment by NilashishC** [comment]
recheck

**Pull request merged**